ROADMAP.md

```diff
@@ -7,26 +7,30 @@ Each release will include necessary tasks that lead to the completion of the sta
 
 ### Workflow Improvements
 
+- Expanding integration of terraform commands :white_check_mark:
+- Expanding validation support :white_check_mark:
 - Detecting initialization outside of VS Code
```

Comment:
```suggestion
- Detecting initialization outside of VS Code :white_check_mark:
```
I believe this actually referred to https://github.com/hashicorp/terraform-ls/pull/385 which was merged and released.